lib/login_page.dart

```diff
@@ -0,0 +1,416 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:provider/provider.dart';
+import 'package:registration_client/app_config.dart';
+import 'package:registration_client/const/utils.dart';
+
+import 'package:registration_client/credentials_page.dart';
+import 'package:flutter/services.dart';
+import 'package:registration_client/provider/app_language.dart';
+import 'package:registration_client/registration_client.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:registration_client/widgets/password_component.dart';
+import 'package:registration_client/widgets/username_component.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({super.key});
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  bool isMobile = true;
+  static const platform =
+      MethodChannel("com.flutter.dev/keymanager.test-machine");
```

Comment:
why is this named as test-machine? instead can we name it as get-machine-keys

Reply:
Renamed it as "io.mosip.get-package-instance"  as only single method channel is being used.